internal/cli/apis.go

```diff
@@ -217,9 +225,16 @@ auth0 apis update <id> --name myapi
 			var current *management.ResourceServer
 
 			if len(args) == 0 {
-				if err := apiID.Ask(cmd, &inputs.ID); err != nil {
+				err := apiID.Pick(cmd, &inputs.ID, cli.apiPickerOptions)
+				if err != nil {
 					return err
 				}
+
+				if inputs.ID == "" {
```

Comment:
Why the nil check if the options function already returns an error if there are no apps?